genesis/genesis_test.go

```diff
@@ -118,6 +118,20 @@ func TestValidateConfig(t *testing.T) {
 			}(),
 			expectedErr: errInitialStakeDurationTooLow,
 		},
+		"locked allocations amount too low": {
```

Comment:
This technically works... I'm not a fan of this existing test though because it tests through `validateConfig` which isn't a part of this package's exported API though. I think a majority of these tests are actually invalid configs, because they would all fail the network id check. Can we test this through the calling code which is `FromFile`? We have an existing test `TestGenesisFromFile` which is set up to do this... we could also use the user-provided genesis config that revealed this bug linked on https://github.com/ava-labs/avalanchego/pull/1173

Reply:
sounds good, i'll move this test to be part of FromFile.